src/constants/index.ts

```diff
@@ -276,20 +277,23 @@ export type PoolsMap = {
 export const POOLS_MAP: PoolsMap = {
   [BTC_POOL_NAME]: {
     name: BTC_POOL_NAME,
+    addresses: BTC_SWAP_ADDRESSES,
     lpToken: BTC_SWAP_TOKEN,
     poolTokens: BTC_POOL_TOKENS,
     addresses: BTC_SWAP_ADDRESSES,
     isSynthetic: true,
```

Comment:
Are the duplicate `addresses` keys intended here?

Reply:
Yeah, it does. Weird why this happens. I guess it's from merging the master.